### PR TITLE
Adds ability to administer Advisers

### PR DESCRIPTION
**Pull request by benlovell**

### CODE REVIEW

File: app/views/admin/advisers/edit.html.erb [hunk 1]
```diff
@@ -15,10 +15,51 @@
...
       <%= f.text_field(:postcode, class: 'form-control') %>
     </div>
 
+    <div class="form-group">
+      <%= f.label :travel_distance %>
+      <%= f.select :travel_distance, TravelDistance.all, { include_blank: true }, { class: 'form-control' } %>
+    </div>
+
+    <div class="form-group">
+      <h4>Accreditations</h4>
```
Comment (benbarnett) [inline app/views/admin/advisers/edit.html.erb]: What was the rationale for `h4` out of interest?

Comment (benlovell) [inline app/views/admin/advisers/edit.html.erb]: I thought they were the most popular `Hx` these days.

Comment (benbarnett) [inline app/views/admin/advisers/edit.html.erb]: Correct

**Comment by benbarnett** [comment]
:+1:

File: app/views/admin/advisers/edit.html.erb (hunk 1 above)
Comment (benlovell) [inline app/views/admin/advisers/edit.html.erb]: :trollface:

**Comment by amansinghb** [comment]
:+1:

**Comment by alexwllms** [comment]
:tada:

**Pull request merged**